ar_exercise/exercise.rb

```diff
@@ -51,6 +114,22 @@ class Apartment < ActiveRecord::Base
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+
+puts "create 3 new apartments"
+Apartment.create do |apt|
+	puts "enter apt address"
+	apt.address = gets.chomp
+	puts "enter monthly rent"
+	apt.monthly_rent = gets.chomp.to_i
+	puts "enter square feet "
+	apt.sqft = gets.chomp.to_i
+	puts "enter num beds "
+	apt.num_beds = gets.chomp.to_i
+	puts "enter num bathrooms"
+	apt.num_baths = gets.chomp.to_i
+end
+
+
```

Comment:
You could then use the variables for the apartments to create tenants that are attached to those apartments. ActiveRecord is super smart and will automatically add the appropriate `apartment_id` for us when we create tenants like this:
```rb
apartment1.tenants.create("some key value pairs for tenant properties, not including apartment_id")
```